core/src/main/java/org/elasticsearch/common/io/stream/StreamInput.java

```diff
@@ -342,6 +342,13 @@ public final double readDouble() throws IOException {
         return Double.longBitsToDouble(readLong());
     }
 
+    public final Double readOptionalDouble() throws IOException {
```

Comment:
I'm sure we have!
On Mar 25, 2016 5:25 PM, "Luca Cavanna" <notifications@github.com> wrote:

> In core/src/main/java/org/elasticsearch/common/io/stream/StreamInput.java
> <https://github.com/elastic/elasticsearch/pull/17351#discussion_r57492947>
> :
>
> > @@ -342,6 +342,13 @@ public final double readDouble() throws IOException {
> >          return Double.longBitsToDouble(readLong());
> >      }
> >
> > +    public final Double readOptionalDouble() throws IOException {
>
> no I mean that I had missed that you had tests in there....I commented too
> early, sorry, it is fine as-is! I wonder if we have missed to add tests to
> BytesStreamTests for similar changes
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/elasticsearch/pull/17351/files/3deac8bf6b6006cc1b966721a6d5804f8fe67333#r57492947>
>